testing-modules/junit-5/src/test/java/com/baeldung/conditional/ConditionalAnnotationsTest.java

```diff
@@ -0,0 +1,119 @@
+package com.baeldung.conditional;
+
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.*;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+public class ConditionalAnnotationsTest {
```

Comment:
Need to rename class to end with "UnitTest" according to guidelines so that the build will pass.